lib/Ohmbrewer_Equipment.h

```diff
@@ -119,19 +120,17 @@ namespace Ohmbrewer {
             /**
              * Constructor
              * @param id The Sprout ID to use for this piece of Equipment
-             * @param pins The list of physical pins this Equipment is attached to
              */
-            Equipment(int id, std::list<int>* pins);
+            Equipment(int id,  int powerPin, int controlPin);
```

Comment:
Equipment won't know about pins (aside from the whichPins interface-y method), so don't include this version of the constructor. Just drop it.